components/layout/UserProfile.js

```diff
@@ -47,7 +46,7 @@ function UserProfile() {
   }
 
   //Decrypt Data:
-  const decryptedData = decryptData(
+  const decryptedData = JSON.parse(
     localStorage.getItem("userData") || localStorage.getItem("mentorData"),
   );
```

Comment:
remove... on login I saved all data in localstorage.. just access that.

Reply:
I am getting what are you saying, local storage is having json stringify object
```
 localStorage.setItem("mentorData", JSON.stringify(res.mentorData));
```
so there is a need to parse it before using.

Reply:
yes yes! you are correct